track.json

```diff
@@ -0,0 +1,8 @@
+{
+    "spotify-card": {
+      "version": "1.5",
+      "remote_location": "https://raw.githubusercontent.com/custom-cards/spotify-card/master/src/spotify-card.js",
```

Comment:
How does these work? Do they download the content to local config directory? If not I think the CDN link I have in the README is a better URL.
Also I think it would be better to have the latest released version instead of master. WDYT?

Reply:
I think what you do now is fine, but some people seem to dislike having a live link.  They would need to manually download the js file the first time and then trigger the custom_updater component to get available updates.  This gives them that option.  As for the link, whichever you prefer.  "Latest" is not something I could test.

Reply:
Worry for the delay. 
This is fine but my question was: if the track.json has version 1.5 the `remote_location` should be `https://github.com/custom-cards/spotify-card/releases/download/v1.5/spotify-card.umd.js`

Reply:
Yes, I agree.  I just used master for testing in my fork.  And maybe the JSON file should be renamed to custom_updater.json to make it more explicit what its purpose is.

Reply:
Sorry. 1.5 in the URL is a problem.  The URL should be stable across versions. Otherwise, the user has to edit it each time the custom_updater won't work.

Reply:
Did you make the necessary changes so we can merge this?

Reply:
Not yet.  I should have some time shortly